apps/www/registry/stories/aspect-ratio.stories.tsx

```diff
@@ -0,0 +1,69 @@
+import Image from "next/image"
+import type { Meta, StoryObj } from "@storybook/react"
+
+import { AspectRatio } from "@/registry/default/ui/aspect-ratio"
+
+/**
+ * Displays content within a desired ratio.
+ */
+const meta: Meta<typeof AspectRatio> = {
+  title: "ui/AspectRatio",
+  component: AspectRatio,
+  tags: ["autodocs"],
+  argTypes: {},
+}
+
+export default meta
+
+type Story = StoryObj<typeof AspectRatio>
+
+/**
+ * The default form of the aspect ratio.
+ */
+export const Default: Story = {
+  render: (args) => (
+    <div className="w-1/2">
+      <AspectRatio {...args} className="bg-slate-50 dark:bg-slate-800">
+        <Image
+          src="https://images.unsplash.com/photo-1576075796033-848c2a5f3696?w=800&dpr=2&q=80"
+          alt="Photo by Alvaro Pinot"
+          fill
+          className="rounded-md object-cover"
+        />
+      </AspectRatio>
+    </div>
+  ),
+  args: {
+    ratio: 16 / 9,
+  },
+}
+
+/**
+ * Use the `1:1` aspect ratio to display a square image.
+ */
+export const Square: Story = {
+  render: Default.render,
```

Comment:
If you're going to re-use the same `render` across every story for a component, you can define it on the `meta` instead.